fleet/api/grpc/server.go

```diff
@@ -0,0 +1,63 @@
+package grpc
+
+import (
+	"context"
+	kitot "github.com/go-kit/kit/tracing/opentracing"
+	kitgrpc "github.com/go-kit/kit/transport/grpc"
+	"github.com/ns1labs/orb/fleet"
+	"github.com/ns1labs/orb/fleet/pb"
+	"github.com/opentracing/opentracing-go"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var _ pb.FleetServiceServer = (*grpcServer)(nil)
+
+type grpcServer struct {
+	pb.UnimplementedFleetServiceServer
+	retrieveAgent kitgrpc.Handler
+}
+
+func NewServer(tracer opentracing.Tracer, svc fleet.Service) pb.FleetServiceServer {
+	return &grpcServer{
+		retrieveAgent: kitgrpc.NewServer(
+			kitot.TraceServer(tracer, "retrieve_agent")(retrieveAgentEndpoint(svc)),
+			decodeRetrieveAgentRequest,
+			encodeAgentResponse,
+		),
+	}
+}
+
+func (gs *grpcServer) RetrievePolicy(ctx context.Context, req *pb.AgentByIDReq) (*pb.AgentRes, error) {
```

Comment:
says `RetrievePolicy` - copy/paste error?

Reply:
Should be RetrieveAgent
Done